src/plugins/auto/utils/config.hpp

```diff
@@ -304,5 +298,6 @@ struct PluginConfig {
     std::map<std::string, std::string> _keyConfigMap;
     const std::set<std::string> _availableDevices =
         {"AUTO", "CPU", "GPU", "TEMPLATE", "MYRIAD", "VPUX", "MULTI", "HETERO", "mock"};
+    const std::set<std::string> _deviceBlacklist = {"GNA", "VPUX"};
```

Comment:
I don't think blacklist is must.  Any clarification about black list?

Reply:
_availableDevices will be removed CVS-90102. Due to the readiness and device support strategy, GNA/VPUX are not in AUTO automatic selection list now, user needs to have them in ov::device::priorities list explicitly if they want to use any of  them.